packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -335,11 +408,10 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
     super.dispose();
   }
 
-  bool get _directionIsXAxis {
-    return widget.direction == DismissDirection.horizontal
-        || widget.direction == DismissDirection.endToStart
-        || widget.direction == DismissDirection.startToEnd;
-  }
+  bool get _directionIsXAxis => switch (widget.direction) {
```

Comment:
The `=>` should be removed here in favor of a code block since it is multiline.

Reply:
[un-resolving this convo, so that hopefully the getter is turned into a block body.]